File: This Is Coding Test/CH 4. Implementation/4-2.js
// 4-2. 시각


function solution(n) {
    var count = 0;

    for (var i = 0; i <= n; i++) {
        for (var j = 0; j < 60; j++) {
            for (var k = 0; k < 60; k++) {

                // 시각에 3이 있는지 확인
                if ((String(i) + String(j) + String(k)).includes("3")) {
                    count = count + 1;
                }
            }
        }
    }

    console.log(count);
}


solution(5)

File: This Is Coding Test/CH 4. Implementation/4-2.py
# 4-2. 시각


def solution(n):
    count = 0

    for i in range(0, n + 1):
        for j in range(0, 60):
            for k in range(0, 60):

                # 시각에 3이 있는지 확인
                if "3" in str(i) + str(j) + str(k):
                    count = count + 1

    print(count)


solution(int(input()))
